src/site/assignments/country-catalogue/country-catalogue.rst

```diff
@@ -53,32 +53,50 @@ Read the assignment description in its entirety before starting.
 Part 1 --- Country
 ==================
 
-You will be creating a class called ``Country`` that will hold some basic data. The data being stored will be the
-country ``name`` (string), ``population`` (int), ``area`` (double), and ``continent`` (string). This class will be used by the
-``CountryCatalogue`` class discussed in Part 2.
+The ``Country`` class will store data related to the country it represents. The data stored will be the  country
+``name``, ``population``, ``area``, and ``continent``. The ``CountryCatalogue`` class described in part 2 will store
+instances of the ``Country`` class.
 
-#. Write a constructor for the ``Country`` class that takes four (4) parameters that will be stored as class fields
-    * String ``name``
-    * int ``population``
-    * double ``area``
-    * String ``continent``
 
-#. Write getter methods for each class field
+#. Write a constructor for the ``Country`` class that takes four (4) parameters to be stored as class fields
 
-#. Write a setter method for the country's ``population``
-    * This will be the only setter we need as the other fields do not typically change all too often for countries
+    * ``name`` --- type ``String``
+    * ``continent`` --- type ``String``
+    * ``population`` -- type ``double``
+    * ``area`` --- type ``double``
 
-#. Write a ``toString`` method that returns a string containing the country ``name``, ``continent``, ``population``, and ``area``, all seperated by commas and a space
-    * Note that you may see some of the area vales represented in scientific notation --- this is OK
-        * Something like 1.234E8
+    .. note::
+
+        Notice that the type for ``population`` is a ``double``. Although it is reasonable to expect that the number of
+        people living in a given country is an integer and does not require decimals, the ``int`` type has a limitation.
+
+        In Java, ``int`` values are stored in four bytes (4B)/32 bits (32b), which means the number of unique values the
+        ``int`` can represent is :math:`2^{32}`, which is :math:`4,294,967,296`. More specifically, ``int`` values can
+        store values between :math:`-2,147,483,648` and :math:`+2,147,483,647`. Since the population of continents can
+        exceed 2 billion, the ``int`` is not going to work.
```

Comment:
If we're going into this, why not simply use `long`? I figured you'd not really mention this at all so agreed with the `double` change. If we have this much detail on `int`s already, the jump to `long` is **much** simpler than jumping to `double`. Floating point comparisons are hell.

Reply:
I had the same thought when writing it out, but went with double since it required less than 0 work since it was already implemented. 

That said, seeing a 2nd :brain: think the same thing seems telling.